transcom/mymove
# INT B-21492 Secondary Transportation Office Assignment Assignable by Admin

## [B-21492](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21492)

## Summary

Allows admin users to assign up to two transportation offices to office users.

Probably under-costed but the follow on task should be pretty straightforward given how much was done for this one.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the admin app.
2. Login as a new or existing admin user.
3. Create a new office user, expect the UI for assigning a transportation office to be slightly different.
4. Test out any combination of one or two transportation office assignments then modify by adding/remove/changing assignments a few times.
5. Ensure the designated primary office is displayed correctly on the list of office user and that the assignments you created display as you created them when you re-open the office user form.
6. Try to break it. The validation message refresh is a bit wonky but that seems to be a quirk of the react-admin library. That shouldn't affect your ability to submit valid data or not submit invalid data.

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
